### PR TITLE
test: publish test results to jenkins

### PR DESCRIPTION
**Pull request by cpanato**

### CODE REVIEW

**Comment by cpanato** [comment]
tap test results for example: https://jenkins-tectonic-installer.prod.coreos.systems/job/tectonic-installer/job/PR-2906/7/tapTestReport/

File: Jenkinsfile [hunk 1]
```diff
@@ -499,6 +499,8 @@ def runRSpecTest(testFilePath, dockerArgs, credentials) {
         throw error
       } finally {
         reportStatusToGithub((err == null) ? 'success' : 'failure', testFilePath, originalCommitId)
+        sh "cat templogfiles/format=tap.log"
```
Comment (mxinden) [inline Jenkinsfile]: Is this line still needed? Looks like a debugging line.

Comment (cpanato) [reply Jenkinsfile]: opppssss.... 😄

Comment (cpanato) [reply Jenkinsfile]: done

**Pull request merged**